packages/block-editor/src/components/image-editor/cropper.js

```diff
@@ -66,7 +66,9 @@ export default function ImageCropper( {
 				crop={ position }
 				zoom={ zoom / 100 }
 				aspect={ aspect }
-				onCropChange={ setPosition }
+				onCropChange={ ( pos ) => {
+					setPosition( pos );
+				} }
```

Comment:
This might not be a necessary change

Reply:
Yes, it's actually a leftover from debugging, I had `console.log('setting pos', pos)` there, but then I decided it's not a bad idea to keep the change. It's not 100% guaranteed that the `onCropChange` callback has just one parameter, and we don't want to pass potential garbage to React's `setState` callback.